documentation/sidebars.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
         'examples/cad-nodefiltering',
         'examples/cad-preview',
         'examples/pointcloud',
+        'examples/clipping',
```

Comment:
```suggestion
        'examples/clipping',
        'examples/pointcloud',
```

not sure about the proper ordering for the examples, but I think it makes sense to separate pointclouds and cad instead of mixing them? Or we might want to have the items AZ-sorted here, what do you think?

Reply:
Yeah, I kinda though clipping would be something generic that worked for both CAD and point cloud but turns out it doesn't work for point clouds yet...